docs/resources/azure_management_group.md

```diff
@@ -0,0 +1,103 @@
+---
+title: About the azure_management_group Resource
+platform: azure
+---
+
+# azure_management_group
+
+Use the `azure_management_group` InSpec audit resource to test properties of an Azure management group.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_management_group` resource block identifies a management group by its `name` or the `resource_id`.
+```ruby
+describe azure_management_group(name: 'abcd-1234') do
+  it { should exist }
+end
+```
+```ruby
+describe azure_management_group(resource_id: '/providers/Microsoft.Management/managementGroups/{groupId}') do
+  it { should exist }
+end
+```
+## Parameters
+
+| Name                           | Description                                                                       |
+|--------------------------------|-----------------------------------------------------------------------------------|
+| name         | Management group name. `20000000-0001-0000-0000-000000000000` |
+| group_id     | Alias for the `name` parameter. |
+| resource_id  | The unique resource ID. `/providers/Microsoft.Management/managementGroups/{groupId}` |
+| expand       | Optional. The `expand: 'children'` includes children in the response. `expand: 'path'` includes the path from the root group to the current group. |
+| recurse      | Optional. The `recurse: true` includes the entire hierarchy in the response. Note that `expand: 'children'` must be passed up if `recurse` is set to `true`. |
```

Comment:
`must be passed up` is this saying that `expand: children` must also be set it `recurse` is set to true?

Reply:
Yes. We can reword if it is not clear.